scripts/import_missing_qc_variants_files.py

```diff
@@ -0,0 +1,124 @@
+import os
+import csv
+import subprocess
+from tqdm import tqdm
+import time
+
+# file constant
+folder_path = '/mnt/disks/data/data-directory/variant_qc/panel_variant_qc'
+output_fname = folder_path + "/combined_sorted_output_file.tsv"
+# Output tsv Gzipped file path
+output_gzipped_tsv = output_fname + ".gz"
+skipped_file_name = 'chrX.removed.add_LCR.add_XPAR.add_failed_filter_awk.tsv'
+
+# List all files in the folder
+def get_files_list():
+    files = []
+    for filename in os.listdir(folder_path):
+        file_path = os.path.join(folder_path, filename)
+        if file_path != output_fname and not skipped_file_name in file_path:
+            files.append(file_path)
+    return files
+
+
+def main_script():
+    filenames = get_files_list()
+    filenames.sort()
+
+    # write the array of objects to a csv file
+    if os.path.exists(output_fname):
+        os.remove(output_fname)  # Overwrite the file with an empty DataFrame
+        print(f"\nFile '{output_fname}' is already exist, so it is cleared first.")
+
+    # read in the files
+    with open(output_fname,"wt",encoding="utf-8") as out_file:
+        # output file header
+        output_file_reader = csv.reader(out_file)
+        output_file_header = None
+        if not os.path.getsize(output_fname) == 0:
+            output_file_header = next(output_file_reader)
+            print(output_file_header)
+        for fname in filenames:
+            with open(fname,"rt",encoding="utf-8") as infile:
+                print(fname)
+                print('Loading data ...')
+                reader = csv.reader(infile, delimiter='\t')
+                for row in reader:
+                    if len(row) == 24:
+                        # check if header is none output file
+                        if row[0] == 'Variant' and output_file_header is None:
+                            row.insert(0, '#chrom')
+                            row.insert(1, 'pos')
+                            final_header = "\t".join(row)+"\n"
+                            out_file.write(final_header)
+                        else:
+                            splited_variant = row[0].split(":")
+                            row.insert(0, splited_variant[0])
+                            row.insert(1, splited_variant[1])
+                            line = f"\t".join(row)+"\n"
+                            out_file.write(line)
+                    else:
+                        print(f"Sorry! The file {fname} does not consist on proper csv columns.")
+                        exit(1)
```

Comment:
Here it would be beneficial to 
1) write out the row contents, so the user has some idea of where the file is faulty
2) maybe raise an exception instead of exiting, so that the user also sees the position in code in which the error was raised.